pkg/statedb/api_handler.go

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package statedb
+
+import (
+	"net/http"
+
+	"github.com/go-openapi/runtime"
+	"github.com/go-openapi/runtime/middleware"
+
+	. "github.com/cilium/cilium/api/v1/server/restapi/statedb"
+	restapi "github.com/cilium/cilium/api/v1/server/restapi/statedb"
+)
+
+func createHandler(db DB) restapi.GetStatedbDumpHandler {
```

Comment:
Let's use a more descriptive name, either s/createHandler/newDumpHandler/, or if we'll use one constructor, then something like `newAPIHandlers`